v2/clients/http/utils/put.go

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/errors"
+)
+
+// Helper method to make the put JSON request and return the body
```

Comment:
This is 99% duplicate of POST. Refactor to one that takes the Method as parameter.

Reply:
Refactored the help method.